requirements.txt

```diff
@@ -4,3 +4,4 @@ matplotlib>=3.0.0
 netCDF4>=1.6.0
 black
 jupyter_black .
+pre-commit
```

Comment:
I don't think  black, jupyter_black or pre-commit should be listed in requirements.txt, 
as these are not really runt-time dependencies, but optional development tools

Reply:
Also, there is this dot after `jupyter_black .`  which makes the file invalid I think (unless this is for some feature I don't know)

Reply:
Removed these from the requirements file